src/crypto/random.js

```diff
@@ -95,41 +97,33 @@ export default {
   },
 
   /**
-   * Create a secure random big integer of bits length
-   * @param {Integer} bits Bit length of the MPI to create
-   * @return {BigInteger} Resulting big integer
+   * Create a secure random MPI that is greater than or equal to min and less than max.
+   * @param {module:type/mpi} min Lower bound, included
+   * @param {module:type/mpi} max Upper bound, excluded
+   * @return {module:BN} Random MPI
    */
-  getRandomBigInteger: function(bits) {
-    if (bits < 1) {
-      throw new Error('Illegal parameter value: bits < 1');
+  getRandomBN: function(min, max) {
+    if (max.cmp(min) <= 0) {
+      throw new Error('Illegal parameter value: max <= min');
     }
-    const numBytes = Math.floor((bits + 7) / 8);
 
-    let randomBits = util.Uint8Array2str(this.getRandomBytes(numBytes));
-    if (bits % 8 > 0) {
-      randomBits = String.fromCharCode(((2 ** (bits % 8)) - 1) &
-        randomBits.charCodeAt(0)) +
-        randomBits.substring(1);
-    }
-    const mpi = new type_mpi(randomBits);
-    return mpi.toBigInteger();
-  },
+    let r;
+    const modulus = max.sub(min);
```

Comment:
Why "modulus"?

Reply:
Yeah, 'diff' or 'difference' would be better.